### ISSUE TITLE
On Linux, build_linux in loadp2 fails

### ISSUE THREAD

**Issue by jomoengineer** [totalspectrum/flexgui#7]
When building loadp2 on Ubuntu 18.04 it fails to find MainLoader.h
```
$ ./build_linux 
loadp2.c:71:10: fatal error: MainLoader.h: No such file or directory
 #include "MainLoader.h"
          ^~~~~~~~~~~~~~
compilation terminated.
```
I was not able to locate this file in the code that was brought down.

**Comment by jomoengineer** [comment]
I posted this on the Parallax forum, but : 
I found I had to build spin2cpp first then create a 'bin' folder under flexgui, copy the executable files in the spin2cpp/build folder to flexgui/bin, set my PATH to include the flexgui/bin folder and then I was able to build loadp2. I copied the resulting build/loadp2 to flexgui/bin.

I also had to copy the spin2cpp 'include' folder and contents to the flexgui folder to compile a .c file.

It compiles now.
